frontend/benefit/applicant/src/components/applications/forms/application/deMinimisAid/useDeminimisAid.ts

```diff
@@ -63,7 +63,9 @@ const useDeminimisAid = (data: DeMinimisAid[]): UseDeminimisAidProps => {
         ...prevDeMinimisAids,
         {
           [DE_MINIMIS_AID_KEYS.GRANTER]: formik.values.granter,
-          [DE_MINIMIS_AID_KEYS.AMOUNT]: parseFloat(formik.values.amount),
+          [DE_MINIMIS_AID_KEYS.AMOUNT]: parseFloat(
+            formik.values.amount.replace(',', '.')
+          ),
```

Comment:
And here? :)

Reply:
they are not the same. see previous comment